.github/workflows/cargo-tests.yml

```diff
@@ -54,6 +54,9 @@ jobs:
       - name: install rust toolchain
         uses: actions-rust-lang/setup-rust-toolchain@v1
 
+      - name: Add required rustup component
+        uses: rustup component add rust-src 
+
```

Comment:
Any explanation on why these are required?

Reply:
Ok found it: https://github.com/OpenZeppelin/polkadot-runtime-template/actions/runs/7454542903/job/20282042449

Probably due to new changes 👍